### ISSUE TITLE
Suggesting preview for security should mention accessibility

### ISSUE THREAD

**Issue by chaals** [w3c/web-share#264]
The security considerations for the [`share` method](https://www.w3.org/TR/2022/PR-web-share-20221215/#share-method) include the following suggestion:

> Content that is previewed and authorized by a user might be safe to forward, however it is not always possible for a person to identify when information should be confidential, so forwarding any content presents a risk. In particular, the [title](https://www.w3.org/TR/2022/PR-web-share-20221215/#dom-sharedata-title) might be used by an attacker to trick a user into misinterpreting the nature of the content.

As well as pointing out that `title` can be used to mislead, it would make sense to point to the accessibility considerations, and/or note that if such preview methods are not designed for high quality accessibility the proposed mitigation of security will fail.

**Comment by marcoscaceres** [comment]
Expanded on the accessibility of the share sheet. https://github.com/w3c/web-share/pull/270 ... @chaals, can you take a look and let me know if what I added addresses the issue? 

Unfortunately, the share sheet is often outside the scope of the browser (i.e., OS provided widget), so the user agent might have any influence or control over the presentation and accessibility aspects of the share sheet. Irrespective, it's helpful to list these guidelines as hopefully OS engineers + OS UX/UI designers will also hopefully read them...
